src/report.tex

```diff
@@ -218,15 +218,18 @@ \section*{Executive Summary}
 contain prompt photon doses which are not present following machine shutdown. It
 does not contain neutrons that are the product of decay reactions, such as neutron
 production from activated water, which is a small component compared to the plasma
-source. Although it is not possible to rigorously asses the uncertainty in a calculation 
+source. Although it is not possible to rigorously assess the uncertainty in a calculation 
 of this scope, the uncertainty has been estimated as a factor of 2. That is, 
 the actual results are expected to be between 0.5 times and 2 times the simulated results.
+This estimate includes uncertainty in neutron transport, activation, photon transport and
+those due to the resolution of the activation mesh. 
```

Comment:
I think they want a statement that clarifies that we believe that the error in absolute result do not undermine our understanding of the relative amounts.  If taken to the extreme, a factor of 2 could mean that the B4C result is higher than the no B4C result.  I don't think that's the case and we need to clarify that.

I think we can argue that many of the systematic errors accumulate in the same way in both cases such that error in the ratio between these results is less than a factor of 2.